ejb3/src/test/resources/org/jboss/as/ejb3/subsystem/subsystem-ejb3-transform.xml

```diff
@@ -3,7 +3,7 @@
         <stateless>
             <bean-instance-pool-ref pool-name="slsb-strict-max-pool"/>
         </stateless>
-        <stateful default-access-timeout="${prop.default-access-timeout:5000}" cache-ref="distributable" clustered-cache-ref="distributable"/>
+        <stateful default-access-timeout="${prop.default-access-timeout:5000}" cache-ref="distributable" clustered-cache-ref="distributable" passivation-disabled-cache-ref="simple"/>
```

Comment:
We need logic added to Ejb3TransformersTestCase that validates this transformation.  Can you add this?

Reply:
@pferraro Sorry, would you mind being more specific about what do you need exactly? As far as I understand, transformer test framework is already testing this attribute. 
The transformer test framework creates the model for the target version using the operation transformer. Later the model is validated using the resource transformer, if the models match, then the transformation is supposed to be fine. So, in essence, if we include the three attributes in the subsystem xml, we will test the rename for all of them. WDYT?

I would like to add a small note here that might require a closer look, it looks like we are forced to include `passivation-disabled-cache-ref="simple"` in this scenario, I mean, the scenario where we are using a subsystem configuration including the deprecated attribute `clustered-cache-ref="distributable"`.

By default, if we use `clustered-cache-ref`, its value is used to initialize the `passivation-disabled-cache-ref` https://github.com/wildfly/wildfly/blob/341bce7d37617fbad19a5e2b44dc60fff9b383fd/ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemAdd.java#L208

I think this is also a corner case for the test framework, because if we don't define `passivation-disabled-cache-ref` in this scenario, the test will never pass due to how test framework is validating the models; matching the model created with the operation transform against the model created with the resource transform.

A different solution would be to use only this pair of attributes: `cache-ref="distributable" passivation-disabled-cache-ref="simple"`, but doing so, we won't be testing the rename using all attributes, that's the reason I included all of them.

Reply:
@yersan What I mean is, I don't think there is anything logic in Ejb3TransformersTestCase that specifically checks that the transformation results for these attributes is correct - only that the transformation results in a valid/consistent model.

Reply:
Hi @pferraro, I have updated the test coverage to test the transformation executing write/read/undefine operations, let me know what do you think.